src/components/TodoInfo/index.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line
```

Comment:
why do you need it (i.e. try to figure out with eslint rule, or ask about that in the chat)